aya/src/programs/cgroup_sysctl.rs

```diff
@@ -0,0 +1,155 @@
+use std::{
+    hash::Hash,
+    os::unix::prelude::{AsRawFd, RawFd},
+};
+
+use crate::{
+    generated::{bpf_attach_type::BPF_CGROUP_SYSCTL, bpf_prog_type::BPF_PROG_TYPE_CGROUP_SYSCTL},
+    programs::{
+        define_link_wrapper, load_program, FdLink, Link, OwnedLink, ProgAttachLink, ProgramData,
+        ProgramError,
+    },
+    sys::{bpf_link_create, bpf_prog_attach, kernel_version},
+};
+
+/// A program used to hook for sysctl.
+/// [`CgroupSysctl`] programs can be attached to a cgroup and will be called every
+/// time a process inside that cgroup tries to read from or write to sysctl knob in proc.
+///
+///
+/// [cgroup]: https://man7.org/linux/man-pages/man7/cgroups.7.html
```

Comment:
I don't think this is referenced by anything?

Reply:
You are correct. Removed it.